lib/core/runtime_v2.nim

```diff
@@ -78,6 +78,13 @@ proc nimRawDispose(p: pointer) {.compilerRtl.} =
 proc nimDestroyAndDispose(p: pointer) {.compilerRtl.} =
   let d = cast[ptr PNimType](p)[].destructor
   if d != nil: cast[DestructorProc](d)(p)
+  when false:
```

Comment:
Was this some debugging that was left in accidentally?

Reply:
Yeah but I'm sure it'll be useful again soon enough. :P